config/default.js

```diff
@@ -0,0 +1,79 @@
+// CONFIG defaults (aka PRODUCTION)
```

Comment:
On other projects we've made development the default. I guess since this is still no-configuration-needed for development having production default is cool?